siriuspy/siriuspy/clientconfigdb/configdb_client.py

```diff
@@ -64,8 +64,13 @@ def get_nrconfigs(self):
         return self._make_request(stats=True)['count']
 
     def get_config_types(self):
-        """Get all configuration types."""
-        return self._make_request()
+        """Get configuration types existing as database entries."""
+        return self._request()
```

Comment:
@xresende, don't you have to use `self._make_request` here?

Reply:
sure! I must have been too quick (and wrong) in using ms code's autocompletion... thanks.

Reply:
fixed it.